lib/tictactoe/Game.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Game do
+  def create(game_id, opponent_type) do
+    GameRound.start(game_id, opponent_type)
+  end
+
+  def make_move_against_bot(round, taken_space) do
+    first_player = round.round_setup.first_player
+    second_player = round.round_setup.second_player
+    board = round.board
+
+    {round_state, board} = Board.apply_move(board, taken_space, first_player.code)
+    winner = round_state == :PlayerWon && :PLAYER_1 || nil
```

Comment:
I am not 100% sure if a bare atom will be truthy?
Is that `((round_state == :PlayerWon) && :Player_1) || nil`?
It reads pretty weird to me.

Reply:
It's an attempt to implement a ternary expression, also not sure how to deal with it properly

Reply:
what about a small case statement?
```
round_state = :PlayerWon

winner = case round_state do
           :PlayerWon -> :PLAYER_1
           _ -> nil
          end

IO.puts winner
```

Reply:
Compressed it into this kind of function
```

  def define_winner(:PlayerWon), do: :PLAYER_1
  def define_winner({:PlayerWon, :PLAYER_1}), do: :PLAYER_1
  def define_winner({:PlayerWon, nil}), do: :PLAYER_2
  def define_winner({_, _}), do: nil
  def define_winner(_), do: nil
```

So the usage in code will be still a one line